internal/provider/incident_workflow_resource.go

```diff
@@ -287,19 +212,41 @@ func (r *IncidentWorkflowResource) Update(ctx context.Context, req resource.Upda
 		return
 	}
 
+	onceFor := []string{}
+	for _, v := range data.OnceFor {
+		onceFor = append(onceFor, v.ValueString())
+	}
+
+	runsOnIncidentModes := []client.UpdateWorkflowRequestBodyRunsOnIncidentModes{}
+	for _, v := range data.RunsOnIncidentModes {
+		runsOnIncidentModes = append(runsOnIncidentModes, client.UpdateWorkflowRequestBodyRunsOnIncidentModes(v.ValueString()))
+	}
```

Comment:
nit nit we _could_ extract these two conversions into funcs and re-use them across update and create.

Reply:
You can't, because the create and update types are different sadly